iina/KeyCodeHelper.swift

```diff
@@ -255,28 +267,40 @@ class KeyCodeHelper {
     return utf8View.count == 1 && utf8View.first! > 32 && utf8View.first! < 127
   }
 
+  static func escapeReservedMpvKeys(_ keystrokesString: String) -> String {
+    // "#" and " " are not valid for `rawKey` because are reserved as tokens when parsing the conf file.
+    // Try to help the user out a little bit before rejecting
+    var keystrokes = keystrokesString.trimmingCharacters(in: .whitespaces)
+    if keystrokes == " " {
+      keystrokes = "SPACE"
+    } else if keystrokes == "#" {
+      keystrokes = "SHARP"
+    }
+    return keystrokes
+  }
+
   static func mpvKeyCode(from event: NSEvent) -> String {
     var keyString = ""
     let keyChar: String
     let keyCode = event.keyCode
     var modifiers = event.modifierFlags
 
     if let char = event.charactersIgnoringModifiers, isPrintable(char) {
+      // Is a classic ASCII printable char.
       keyChar = char
-      let (_, rawKeyChar) = event.readableKeyDescription
-      if rawKeyChar != char {
-        modifiers.remove(.shift)
-      }
+      // The char in `charactersIgnoringModifiers` will be either uppercase or lowercase,
```

Comment:
From the Swift DocC section [Add a Basic Description for Each Symbol](https://www.swift.org/documentation/docc/writing-symbol-documentation-in-your-source-files#Add-a-Basic-Description-for-Each-Symbol):
> If a symbol already has a source comment that begins with two forward slashes (//), insert an additional forward slash (/) to convert it to a documentation comment.

I'm pretty sure [Markdown formatting](https://www.swift.org/documentation/docc/formatting-your-documentation-content) can only be used in swift documentation comments. I'm having trouble finding a reference, but I believe documentation comments will only be recognized before declarations, not inlined in a code block.